includes/class-amp-theme-support.php

```diff
@@ -1657,41 +1642,87 @@ public static function ensure_required_markup( DOMDocument $dom, $script_handles
 			}
 		}
 
-		/*
-		 * "Specify the <script> tags for render-delaying extensions (e.g., amp-experiment, amp-dynamic-css-classes, and amp-story)."
-		 * "Specify the <script> tags for remaining extensions (e.g., amp-bind, ...). These extensions are not render-delaying and therefore
-		 * should not be preloaded because they might take away important bandwidth for the initial render."
-		 * {@link https://docs.google.com/document/d/169XUxtSSEJb16NfkrCr9y5lqhUR7vxXEAsNxBzg07fM/edit AMP Hosting Guide}
-		 */
+		// "5. Load the AMP runtime."
 		if ( isset( $amp_scripts['amp-runtime'] ) ) {
 			$ordered_scripts['amp-runtime'] = $amp_scripts['amp-runtime'];
```

Comment:
Is this missing an `unset`? It looks like it will be merged in a second time a bit further down below otherwise.
```suggestion
			$ordered_scripts['amp-runtime'] = $amp_scripts['amp-runtime'];
			unset( $amp_scripts['amp-runtime'] );
```